database/config/config.js

```diff
@@ -1,28 +1,28 @@
-const keys = require("../../utils/keys");
+const keys = require('../../utils/keys');
 
 const config = {
   development: {
-    username: "root",
+    username: 'root',
```

Comment:
Private Informations like username, password, ... Should be stored in the .env file. Then you can import the env and access those variables.